packages/liveblocks-react/scripts/check-exports.ts

```diff
@@ -19,6 +19,19 @@ const ALLOW_DIFFERENT_JSDOCS = [
   "useUser",
 ];
 
+// These exports may exist at the top-level without a factory equivalent
+const ALLOW_NO_FACTORY = [
+  "ClientSideSuspense",
+  "createLiveblocksContext", // the factories themselves, obviously
+  "createRoomContext", // the factories themselves, obviously
+  "Json",
+  "JsonObject",
+  "MutationContext",
+  "shallow",
+  "UseThreadsOptions",
+  "useClient", // XXX Should we export this from the factory too, for consistency? 🤔
```

Comment:
Maybe this is actually a valid complaint, will look into it tomorrow. Maybe it's nice to also export this new hook from the factory!

Reply:
Fixed in 9073bffd2493cff90bde1c903f98ac8a1a10498d.